### PR TITLE
closes #1530_Doc: Updated R version in multiple files

### PR DESCRIPTION
**Pull request by sadchla-codes**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/main/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/main/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/main/articles/programming_strategy.html#deprecation-1)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by sadchla-codes** [comment]
Not 100% sure what happened with this pull request as it was inaccurately showing that I was the author on multiple other files changes. Will re-clone the directory and work on a new and separate branch.

File: vignettes/faq.Rmd [hunk 1]
```diff
@@ -49,8 +49,8 @@ knitr::opts_chunk$set(
...
+* The choice of R Version is not set in stone.  However, a common development environment is important to establish when working across multiple companies and multiple developers. We only use the three latest R versions and the closest date of snapshots of R packages available when that R version came out. This need for a common development environment also carries over for our choice of package versions.  
+* GitHub allows us through the Actions/Workflows to test `{admiral}` under several versions of R as well as several versions of dependent R packages needed for `{admiral}`.  Currently we test `{admiral}` against R Version 4.0 with a CRAN package snapshot from 2021-03-31 and the latest R version with the latest snapshots of packages.  You can view this workflow and others on our [admiralci GitHub Repository](https://github.com/pharmaverse/admiralci).
```
Comment (bms63) [inline vignettes/faq.Rmd:53]:
```suggestion
* GitHub allows us through the Actions/Workflows to test `{admiral}` under several versions of R as well as several versions of dependent R packages needed for `{admiral}`.  Currently we test `{admiral}` against the three latest versions of R with the closest snapshot of CRAN packages to the R version.  You can view this workflow and others on our [admiralci GitHub Repository](https://github.com/pharmaverse/admiralci).
```

**Review by bms63 — CHANGES_REQUESTED** [review]
Little wordsmithing

**Comment by sadchla-codes** [comment]
@bms63 I'm running to the issue below when I try to update documentation, is that similar to what you got yesterday?
documentation issue

File: vignettes/faq.Rmd (hunk 1 above)
Comment (sadchla-codes) [reply vignettes/faq.Rmd]: Change implemented!

File: vignettes/faq.Rmd [hunk 2]
```diff
@@ -49,8 +49,8 @@ knitr::opts_chunk$set(
...
-* GitHub allows us through the Actions/Workflows to test `{admiral}` under several versions of R as well as several versions of dependent R packages needed for `{admiral}`.  Currently we test `{admiral}` against R Version 3.6.3 with a CRAN package snapshot from 2020-02-29, R Version 4.0 with a CRAN package snapshot from 2021-03-31 and the latest R version with the latest snapshots of packages.  You can view this workflow and others on our [admiralci GitHub Repository](https://github.com/pharmaverse/admiralci).
+* The choice of R Version is not set in stone.  However, a common development environment is important to establish when working across multiple companies and multiple developers. We only use the three latest R versions and the closest date of snapshots of R packages available when that R version came out. This need for a common development environment also carries over for our choice of package versions.
```
Comment (thomas-neitmann) [inline vignettes/faq.Rmd:52]: This paragraph refers to our dev environment which is only a single R version. We should only reference the 3 version we test on in the paragraph below.

**Review by thomas-neitmann — CHANGES_REQUESTED** [review]

File: vignettes/faq.Rmd (hunk 2 above)
Comment (bms63) [reply vignettes/faq.Rmd:52]: Sorry he is saying that as developers we only work in 1 R version.  The earliest of the 3 latest R versions.  So 4.0 is earliest of 4.0, 4.1 and 4.2

Comment (bms63) [reply vignettes/faq.Rmd:52]: We test against 3 R versions

**Review by bms63 — APPROVED** [review]
Triggering workflow

**Comment by sadchla-codes** [comment]
@bms63  @thomas-neitmann  Need a little guidance on how to proceed with this one because it looks like closed PR https://github.com/pharmaverse/admiral/pull/1567#issue-1438469499 
addressed the same issue.

**Comment by bms63** [comment]
AH shoot..  I thought I was going crazy.  I had no idea you both were working on the same issue.  I thought it was in differnt places in the documentation.  I think we can just close this out if this was the only update in the faqs.

Sorry @sadchla-codes !!

**Pull request closed without merging**